src/Microsoft.Identity.Web.UI/Microsoft.Identity.Web.UI.csproj

```diff
@@ -23,4 +23,8 @@
     <ProjectReference Include="..\Microsoft.Identity.Web\Microsoft.Identity.Web.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
?

Reply:
I think this was a vestige of when I was trying to resolve the errors associated with IdWeb.UI target frameworks. Removing!

Reply:
Hmm, sorry. Looks like this is necessary. Without: 
![image](https://github.com/AzureAD/microsoft-identity-web/assets/69649063/db9ffa39-7653-4dea-9b67-173278ee046e)